src/Microsoft.AspNet.SignalR.SqlServer/SqlInstaller.cs

```diff
@@ -53,7 +53,8 @@ private bool IsSqlEditionSupported(string connectionString)
             var operation = new DbOperation(connectionString, "SELECT SERVERPROPERTY ( 'EngineEdition' )", _trace);
             var edition = (int)operation.ExecuteScalar();
 
-            return edition >= SqlEngineEdition.Standard && edition <= SqlEngineEdition.Express;
+            return edition >= SqlEngineEdition.Standard && edition <= SqlEngineEdition.Express ||
```

Comment:
```suggestion
            return (edition >= SqlEngineEdition.Standard && edition <= SqlEngineEdition.Express) ||
```

I think the order of operations is correct without parens, but we like to be extra clear.

Reply:
Changes applied as suggested, thank you.